bin/dyn_db_hack.ml

```diff
@@ -0,0 +1,55 @@
+(* This module automatically discovers and loads database
```

Comment:
Does this really need to be called `dyn_db_hack`? It may well be a hack, and that's fine (although it doesn't look too egregious to me) but I'd prefer a different filename.

Reply:
Fixed (modulo the renaming of `dyn_db_hack.ml`).

It is a hack. Once a principal solution emerges this hack should be removed.